config/config.go

```diff
@@ -112,6 +112,7 @@ func (c *Config) GetDatabaseParams() dbparams.Database {
 		MaxOpenConnections:    viper.GetInt("database.max_open_connections"),
 		MaxIdleConnections:    viper.GetInt("database.max_idle_connections"),
 		ConnectionMaxLifetime: viper.GetDuration("database.connection_max_lifetime"),
+		DisableAutoMigrate:    viper.GetBool("database.disable_auto_migrate"),
```

Comment:
@nopcoder fine with this - but it needs to be documented.
I recommend setting an explicit default value as well.

This seems to be the case for all other dbparams except for connection string.

Reply:
Updated the docs. Note that the defaults are set by the 'db' package by `normalizeDBParams` function

Reply:
great, thanks!

Reply:
Thanks you!